crates/iota-graphql-rpc/src/types/event/mod.rs

```diff
@@ -269,12 +268,7 @@ impl Event {
         stored: StoredEvent,
         checkpoint_viewed_at: u64,
     ) -> Result<Self, Error> {
-        let Some(Some(sender_bytes)) = ({
-            #[cfg(feature = "postgres-feature")]
-            {
-                stored.senders.first()
-            }
-        }) else {
+        let Some(Some(sender_bytes)) = ({ stored.senders.first() }) else {
```

Comment:
Nit: Simplification
```suggestion
        let Some(Some(sender_bytes)) = stored.senders.first() else {
```

Reply:
Done in f501d28d9e0562a4f3fbc0f8e2f98fd291659c2d